### DISCUSSION TITLE
How to import once and use it globally?

### DISCUSSION THREAD

**Discussion by mariusa** [vueuse/vueuse#457] [Q&A] [answered]
Hi,

Would you please add to https://vueuse.org/guide.html
how to import all/specific helpers once and use it globally?

For example with `get/set`, I'd like to be able to use in any view `set(name, 'santa')` without importing `set` every time.

I've tried this in main.ts but doesn't work:
```
import { get, set } from '@vueuse/core'
import { createApp } from 'vue'
import App from './App.vue'

const app = createApp(App).mount('#app')
app.component('set', set)
```

Thanks!

**Comment by antfu** [comment, marked as answer]
Tho I would not recommend doing this, but you can:

```ts
import { get, set } from '@vueuse/core'

window.get = get
window.set = set
```

**Reply by mariusa** [reply]
Thanks :)

**Reply by ridsgn** [reply]
hi @antfu why its not recommended ? I want to implement this approach for `useIdle` too
